common/views/components/CataloguePageLayout/CataloguePageLayout.tsx

```diff
@@ -1,7 +1,7 @@
 import { FunctionComponent, useEffect, useState } from 'react';
 import PageLayout, { Props as PageLayoutProps } from '../PageLayout/PageLayout';
 import InfoBanner from '../InfoBanner/InfoBanner';
-import SignIn from '@weco/catalogue/components/SignIn/SignIn';
+import SignIn from '../SignIn/SignIn';
 import { prefix } from '@weco/identity/src/utility/prefix';
```

Comment:
Bikeshedding - are we sharing code between all apps now? Seems a whiff of coupling which is always a little dangerous/